apps/worker/src/app/workflow/workflow.module.ts

```diff
@@ -110,6 +116,17 @@ const SERVICES = [
     provide: TriggerQueueService,
     useClass: TriggerProcessorQueueService,
   },
+  {
+    provide: WsQueueService,
+    useClass: WsQueueService,
+  },
```

Comment:
Why is this needed here?

Reply:
The `WsQueueService` doesn't have the worker logic and yes it's to get the metrics